osu.Game/Screens/Play/HUD/ArgonSongProgress.cs

```diff
@@ -114,6 +118,7 @@ protected override void Update()
             base.Update();
             content.Height = bar.Height + bar_height + info.Height;
             graphContainer.Height = bar.Height;
+            Colour = AccentColour.Value;
```

Comment:
I question why this needs to be in update. Let's use the bindable as a bindable instead yeah?

Reply:
jesus. not sure how i'd missed that.

Reply:
removed in https://github.com/ppy/osu/pull/29936/commits/9f4e48dde78eb1b76c6131999aa21aeb4dc42843. also found one more when i was looking at that again.